scalahost/sbt/src/main/scala/scala/meta/scalahost/sbt/ScalahostSbtPlugin.scala

```diff
@@ -22,15 +40,21 @@ object ScalahostSbtPlugin extends AutoPlugin {
 
   lazy val scalahostAllSettings =
     scalahostBaseSettings ++
-      scalahostInjectCompilerPluginSettings ++
+      inTestAndCompile(scalahostConfigSettings) ++
+      scalahostInjectScalacOptions ++
+      scalahostInjectDependency ++
       scalahostHasMirrorSettings
 
   lazy val scalahostBaseSettings: Seq[Def.Setting[_]] = Def.settings(
     ivyConfigurations += Scalameta,
+    scalametaSourcepath := baseDirectory.in(ThisBuild).value,
     resolvers += Resolver.bintrayRepo("scalameta", "maven")
   )
+  lazy val scalahostConfigSettings: Seq[Def.Setting[_]] = Def.settings(
+    scalametaSemanticdb := ScalametaSemanticdb.Fat
```

Comment:
Or rather, the plugin should be `ScalametaPlugin` named instead of `ScalahostSbtPlugin`?

This also opens the question if we should rename `sbt-scalahost` to `sbt-scalameta`. Having explained what were doing to several people, I have noticed that the word "scalahost" is confusing to many.